src/app/zap-templates/zcl/data-model/chip/thermostat-cluster.xml

```diff
@@ -84,16 +205,16 @@ limitations under the License.
       <access op="read" privilege="view"/>
       <access op="write" privilege="manage"/>
     </attribute>
-    <attribute side="server" code="0x001C" define="SYSTEM_MODE" type="ENUM8" min="0x00" max="0x07" writable="true" default="0x01" optional="false">
+    <attribute side="server" code="0x001C" define="SYSTEM_MODE" type="ThermostatSystemMode" min="0x00" max="0x07" writable="true" default="0x01" optional="false">
       <description>SystemMode</description>
       <access op="read" privilege="view"/>
       <access op="write" privilege="manage"/>
     </attribute>
-    <attribute side="server" code="0x001E" define="THERMOSTAT_RUNNING_MODE" type="ENUM8" min="0x00" max="0x04" writable="false" optional="true">ThermostatRunningMode</attribute>
-    <attribute side="server" code="0x0020" define="START_OF_WEEK" type="ENUM8" min="0x00" max="0x06" writable="false" optional="true">StartOfWeek</attribute>
+    <attribute side="server" code="0x001E" define="THERMOSTAT_RUNNING_MODE" type="ThermostatRunningMode" min="0x00" max="0x04" writable="false" optional="true">ThermostatRunningMode</attribute>
```

Comment:
Would this not get some form of namespace overlap because type is the same name as the variable?

Should we standardise on some suffix for types? the spec copy I have just says "enum8" without actually explaining the values or name the type.

Reply:
Maybe this should be fixed in spec and not have ThermostatCluster have things with prefix `Thermostat` because that stutters.

Then again that does not fix `StartOfWeek`. Maybe at least the enum there should be `ThermostatStartOfWeek`

Reply:
> the spec copy I have just says "enum8" without actually explaining the values or name the type.

Right the spec needs to define these names before we start shipping them in the SDK.